crates/router/src/connector/authorizedotnet/transformers.rs

```diff
@@ -904,7 +929,14 @@ impl From<SyncStatus> for enums::RefundStatus {
         match transaction_status {
             SyncStatus::RefundSettledSuccessfully => Self::Success,
             SyncStatus::RefundPendingSettlement => Self::Pending,
-            _ => Self::Failure,
+            SyncStatus::AuthorizedPendingCapture
+            | SyncStatus::CapturedPendingSettlement
+            | SyncStatus::SettledSuccessfully
+            | SyncStatus::Declined
+            | SyncStatus::Voided
+            | SyncStatus::CouldNotVoid
+            | SyncStatus::GeneralError
+            | SyncStatus::FDSPendingReview => Self::Failure,
```

Comment:
Why FDSPendingReview, SettledSuccessfully, AuthorizedPendingCapture, CapturedPendingSettlement, Voided should be mapped to Failure?

Reply:
@deepanshu-iiitu  will handle the status mapping modifications in the pull request where he is addressing the connector audit fixes.